packages/backend/src/submissions/submissions.service.ts

```diff
@@ -11,7 +11,7 @@ export class SubmissionsService {
 
   async create(data: CreateSubmissionDto): Promise<SubmissionEntitiy> {
     try {
-      return await this.prisma.submission.create({ data })
+      return await this.prisma.submission.create({ data: { ...data, status: 'IN_REVIEW' } })
```

Comment:
Is this, how you suppose to add a new field to an existing object?

Reply:
yes
btw it's better to use the enum value (`SUBMISSION_STATUS.IN_REVIEW`) instead of typing it as a string